### ISSUE TITLE
Clock + Daily (Horizontal) lacking subtitle option

### ISSUE THREAD

**Issue by Dogway** [breezy-weather/breezy-weather#1324]
### Steps to reproduce

Set the widget style to Clock + Daily (Horizontal)

### Expected behavior

I would expect to show a subtitle option to add more custom details

### Actual behavior

It simply shows a minimal widget without the option for subtitles.

### Weather source used

Open-Meteo

### Breezy Weather version

5.2.5

### Android version

Android 14

### Device

Sony Xperia 10 IV

### Other details

![Open](https://github.com/user-attachments/assets/e283c656-6594-4a1f-9a17-a0b5bdea7a6d)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Not a bug; there is indeed no such option on this widget

**Comment by Dogway** [comment]
> Not a bug; there is indeed no such option on this widget

Exactly, it's an UX flaw. You would expect the same features in horizontal and vertical, inconsistency at best.
This refrains me from fully switching to this app, while still keeping Transparent Weather.

**Comment by papjul** [comment]
No, not all widgets have the same options. Some widgets don't have any options at all. It depends on the features and layout (the vertical has a subtitle, while the horizontal doesn't, for example).

Anyway, the custom subtitle will only exist on the text widget + this widget will be removed once #937 is complete, so feel free to keep using Transparent Weather if it fits your needs better ;)
